R/ncdf.R

```diff
@@ -326,7 +398,7 @@ read_ncdf = function(.x, ..., var = NULL, ncsub = NULL, curvilinear = character(
   dims <-meta$dimension[match(meta$axis$dimension[meta$axis$variable == var],
                               meta$dimension$id), ]
 
-  canon_order <- c("X", "Y", "Z", "T")
+  canon_order <- c("X", "Y", "Z", "T")[!is.na(c_v[c("X", "Y", "Z", "T")])]
```

Comment:
This was actually a bug before. Doing this such that a data source could be XYT, XYZ, or XYZT.